Tests/GitHubProjects.tests.ps1

```diff
@@ -0,0 +1,332 @@
+            }
+
+            It 'Name is correct' {
+                $results.name | Should be $defaultRepoProject
+            }
+
+            It 'Description should be updated' {
+                $results.body | Should be $modifiedRepoProjectDesc
+            }
+        }
+    }
+
+    Describe 'Create Project' {
+        Context 'Create User projects' {
+            BeforeAll {
+                $project = @{id = 0}
+            }
+            AfterAll {
+                $null = Remove-GitHubProject -Project $project.id -Confirm:$false
+                Remove-Variable project
+            }
+
+            $project.id = (New-GitHubProject -UserProject -Name $defaultUserProject -Description $defaultUserProjectDesc).id
+            $results = Get-GitHubProject -Project $project.id
+            It 'Project exists' {
+                $results | Should Not BeNullOrEmpty
+            }
+
+            It 'Name is correct' {
+                $results.name | Should be $defaultUserProject
+            }
+
+            It 'Description should be updated' {
+                $results.body | Should be $defaultUserProjectDesc
+            }
+        }
+
+        Context 'Create Organization projects' {
+            BeforeAll {
+                $project = @{id = 0}
+            }
+            AfterAll {
+                $null = Remove-GitHubProject -Project $project.id -Confirm:$false
+                Remove-Variable project
+            }
+
+            $project.id = (New-GitHubProject -OrganizationName $script:organizationName -Name $defaultOrgProject -Description $defaultOrgProjectDesc).id
+            $results = Get-GitHubProject -Project $project.id
+            It 'Project exists' {
+                $results | Should Not BeNullOrEmpty
+            }
+
+            It 'Name is correct' {
+                $results.name | Should be $defaultOrgProject
+            }
+
+            It 'Description should be updated' {
+                $results.body | Should be $defaultOrgProjectDesc
+            }
+        }
+
+        Context 'Create Repo projects' {
+            BeforeAll {
+                $project = @{id = 0}
+            }
+            AfterAll {
+                $null = Remove-GitHubProject -Project $project.id -Confirm:$false
+                Remove-Variable project
+            }
+
+            $project.id = (New-GitHubProject -OwnerName $script:ownerName -RepositoryName $repo.name -Name $defaultRepoProject -Description $defaultRepoProjectDesc).id
+            $results = Get-GitHubProject -Project $project.id
+            It 'Project Exists' {
+                $results | Should Not BeNullOrEmpty
+            }
+
+            It 'Name is correct' {
+                $results.name | Should be $defaultRepoProject
+            }
+
+            It 'Description should be updated' {
+                $results.body | Should be $defaultRepoProjectDesc
+            }
+        }
+    }
+
+    Describe 'Remove Project' -tag remove{
```

Comment:
nit:
```suggestion
    Describe 'Remove Project' -tag remove {
```

Reply:
I removed the tag in this case - I was just using those to be able to test a section at a time.